Riot/Modules/VoiceBroadcast/VoiceBroadcastSDK/VoiceBroadcastInfo.m

```diff
@@ -66,8 +68,13 @@ + (id)modelFromJSON:(NSDictionary *)JSONDictionary
             voiceBroadcastId = relatesTo.eventId;
         }
     }
+    
+    NSInteger lastChunkSequence = -1;
```

Comment:
I think you can use 0 as default value, because the actual first chunk sequence is 1
@yostyle can you confirm?

Reply:
Updated in last commit. The first chunk sequence is 1 by default.